core/shared/src/main/scala/fs2/fs2.scala

```diff
@@ -34,4 +34,10 @@ package object fs2 {
     * Alias for `Nothing` which works better with type inference.
     */
   type INothing <: Nothing
+
+  /**
+    * Alias for `Any` which may work better with type inference.
+    */
+  type IAny >: Any
```

Comment:
If we were to go this way, I don't think we need `IAny` as scalac doesn't treat `Any` special from type inference perspective.